rabbit-core/pom.xml

```diff
@@ -30,47 +30,49 @@
         <dependency>
             <groupId>mysql</groupId>
             <artifactId>mysql-connector-java</artifactId>
-            <version>8.0.23</version>
+            <version>8.0.25</version>
         </dependency>
         <dependency>
             <groupId>org.dom4j</groupId>
             <artifactId>dom4j</artifactId>
             <version>2.1.3</version>
         </dependency>
+        <!-- Note: Apache poi v5.x is current incompatible with the QuickAndDirtyXLSReader-->
         <dependency>
             <groupId>org.apache.poi</groupId>
             <artifactId>poi</artifactId>
-            <version>5.0.0</version>
```

Comment:
Should be okay. Is it not fixable by using another api call in poi 5.0.0?

Reply:
Maybe, but the problem is pretty deep.